ci/run_python_tests.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+set -euo pipefail
+
+test_cmd="${1:-}"
+test_args="${2:-}"
+node_version="${3:-}"
+
+[ "$#" -ne 3 ] && { echo -e "Error: function requires exactly three arguments.\n\n"; exit 1;}
+
+# Functions
+function fn_die() {
+  echo -e "$1" >&2
+  exit "${2:-1}"
+}
+
+function import_gpg_keys() {
+  # shellcheck disable=SC2145
+  printf "%s\n" "Tagged build, fetching keys:" "${@}" ""
+  # shellcheck disable=SC2207
+  declare -r my_arr=( $(echo "${@}" | tr " " "\n") )
+
+  for key in "${my_arr[@]}"; do
+    echo "Importing key: ${key}"
+    gpg -v --batch --keyserver hkps://keys.openpgp.org --recv-keys "${key}" ||
+    gpg -v --batch --keyserver hkp://keyserver.ubuntu.com --recv-keys "${key}" ||
+    gpg -v --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "${key}" ||
+    gpg -v --batch --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "${key}"
+  done
+}
+
+function check_signed_tag() {
+  # Checking if git tag signed by the maintainers
+  if git verify-tag -v "${1}"; then
+    echo "${1} is a valid signed tag"
+    return 0
+  else
+    echo "Git tag = ${1} signature is NOT valid. Codesigning will be skipped..."
+    return 1
+  fi
```

Comment:
Why do we need these two functions check here if it is being checked already under `setup_env.sh` file :thinking:

Reply:
Not checking the same thing, 
in the setup_env checking the artifact of sidechain SDK
In python_tests.py checking the zen artifacts